aten/src/ATen/native/LinearAlgebra.h

```diff
@@ -9,6 +9,7 @@ class Scalar;
 
 namespace at {
 struct TensorIterator;
+struct Tensor;
```

Comment:
Is this OK? I don't see this used anywhere else in the PyTorch source code. And I fear adding `#include <ATen/ATen.h>` header, as compilation fails with an error of a dependency is added. I'll appreciate some guidance here. @IvanYashchuk

Reply:
No, it's not okay. There's `<ATen/Tensor.h>` that you can use. But as mentioned below put these declarations in the sparse related folder, not here.

Reply:
Done. Thanks Ivan :)